library/src/G4CMPKaplanQP.cc

```diff
@@ -98,6 +99,8 @@ void G4CMPKaplanQP::SetFilmProperties(G4MaterialPropertiesTable* prop) {
   if (!(prop->ConstPropertyExists("gapEnergy") &&
         prop->ConstPropertyExists("phononLifetime") &&
         prop->ConstPropertyExists("phononLifetimeSlope") &&
+        prop->ConstPropertyExists("absorberEff") &&
```

Comment:
Take these two lines back out.  We want these parameters to be optional, not required.